src/schema.rs

```diff
@@ -0,0 +1 @@
+infer_schema!("dotenv:DATABASE_URL");
```

Comment:
To view the schema of the current database (what infer_schema! would create) run `diesel print-schema`. I used the result of that command to fix it. Hopefully this schema works for others as well.